programmers/인형뽑기.js

```diff
@@ -0,0 +1,22 @@
+function solution(board, moves) {
+    const result = [];
+    let cnt = 0;
+    moves = moves.map(el => el - 1);
+      
+    for (let move of moves) {
+      for (let j = 0 ; j < board.length; j++) {
+        if (board[j][move] > 0) {
```

Comment:
```suggestion
        if (board[j][move]) {
```
0을 false로 인식하므로 기입하지 않으셔도 됩니다!